src/matlab2tikz.m

```diff
@@ -98,6 +98,9 @@ function matlab2tikz(varargin)
 %   interpret tick labels as TeX. MATLAB(R) doesn't do that by default.
 %   (default: false)
 %
+%   MATLAB2TIKZ('arrowHeadSizeFactor', FLOAT, ...) allows to resize the arrow heads
+%   in quiver plots by rescaling the arrow heads by a positive scalar. (default: 10)
```

Comment:
So i think this is the critial point. In principal you mix two options

1. Should the arrowHeadSize be changed

2. To what value should it be set.

So i would tend to either add a boolean value, that determines whether we should override the default value, or set the default to 0/-1 and only change the default size for arrowHeadSizeFactor>0